shaders/CMakeLists.txt

```diff
@@ -143,7 +143,7 @@ if (PLATFORM_IPHONE)
 endif()
 
 # Do not include glm's CMakeLists.txt, because it's outdated and not necessary.
-target_include_directories(${PROJECT_NAME} PUBLIC ${OMIM_ROOT}/3party/glm)
+target_include_directories(${PROJECT_NAME} PUBLIC ${OMIM_ROOT}/3party/glm ${OMIM_ROOT}/3party/utf8cpp/include)
```

Comment:
Is it needed if we already have cmake/OmimConfig.cmake change?

Reply:
CI builds fail without this. I haven't tracked down entirely, why exactly the `3PARTY_INCLUDE_DIRS` doesn't help here but it doesn't.

Reply:
Otherwise perhaps` ${OMIM_ROOT}/3party/glm` should probably also go into` 3PARTY_INCLUDE_DIRS`?

Reply:
- Ok, for some reason, shaders library doesn't use omim_add_library
- glm is used only in drape, drape_frontend, sharders

Reply:
It would be great to track down _why exactly the 3PARTY_INCLUDE_DIRS doesn't help here_

Reply:
> It would be great to track down _why exactly the 3PARTY_INCLUDE_DIRS doesn't help here_

I think vng has pointed out the direct answer:
The `3PARTY_INCLUDE_DIRS` doesn't help here, because `shaders/CMakeLists.txt` uses `add_library` instead of `omim_add_library` like the rest, and `omim_add_library` pulls in the `3PARTY_INCLUDE_DIRS`.